### ISSUE TITLE
dynamic json modify

### ISSUE THREAD

**Issue by perfectwebtech** [airbnb/lottie-web#3071]
hello

is there way i can able to modify adobe after effect exported json 

insert new or modify text with animation font size color etc ? 

and png or jpeg also ? 

is there how possible can help us for web base then 

littie cli works for convert mp4 but this dynamic template  way i need to make it working if help would be very appreciate

**Comment by alectrocute** [comment]
Yes, obviously you can modify the JSON. Experiment and get back to us.
